dqgen/services/query_template_registry.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python3
+
+# query_template_registry.py
+# Date:  15/09/2021
+# Author: Eugeniu Costetchi
+# Email: costezki.eugen@gmail.com
+
+"""
+    This modules provides an easy access to the SPARQL queries templates from resources
+"""
+from dqgen.adapters.resource_fetcher import get_query_template
+
+
+class QueryTemplateRegistry:
```

Comment:
Very good idea! Encapsulating access to the constants possibly allows us for multiple configurations in the future.

Reply:
needs to be documented, though